microsoft/testsuites/kselftest/kselftest.py

```diff
@@ -92,33 +92,53 @@ def _check_exists(self) -> bool:
         )
 
     def __init__(
-        self, node: Node, kselftest_file_path: str, *args: Any, **kwargs: Any
+        self,
+        node: Node,
+        kselftest_working_path: str,
+        kselftest_file_path: str,
+        *args: Any,
+        **kwargs: Any,
     ) -> None:
         super().__init__(node, *args, **kwargs)
 
         # tar file path specified in yml
+        self._kselftest_working_dir = kselftest_working_path
```

Comment:
`self._kselftest_working_dir` should be like `self._kselftest_working_path`. Don't mix use of path and dir.